tmt/tests/booted/readonly/025-test-restorecon.nu

```diff
@@ -0,0 +1,12 @@
+use std assert
+use tap.nu
+
+tap begin "Run restorecon"
+
+# Run restorecon and capture trimmed output
+let out = (restorecon -vnr /var/ /etc/ /usr/ /boot/ | str trim)
```

Comment:
I think I'd say we should test all of these separately; we'll need to have `/boot` marked as a known issue at least right?

I am a little wary of adding `/var` into this in the general case as labels might be different for at least semi-valid reasons. `/var` is conceptually a big bag of state not owned by us.